game/models.py

```diff
@@ -179,6 +142,21 @@ def save(self, *args, **kwargs):
             self.created_at = timezone.now()
         return super(Task, self).save(*args, **kwargs)
 
+class DeployedTaskImage(models.Model):
+    task = models.ForeignKey(Task, related_name='deployed_images')
```

Comment:
https://github.com/Gfif/blackbox3/blob/710ce3e5739964ef92220c8c4192de556daf7992/game/models.py#L135-L136
Here is the answer.